tests/main/interfaces-fuse_support/task.yaml

```diff
@@ -57,8 +57,13 @@ execute: |
     test-snapd-fuse-consumer.create $MOUNT_POINT
     # use "[f]oo" to search for "foo" in ps output without having to filter yourself out
     PID=$(ps aux | awk '/[t]est-snapd.*create/{print $2}')
+    match=0
+    # there may be more than one PID that matched, at least one should be the right match
     for p in $PID; do
-        echo "$p command: $(cat /proc/${p}/cmdline | tr '\0' ' ')"
+        echo "checking process $p, command: $(cat /proc/${p}/cmdline | tr '\0' ' ')"
+        if  cat /proc/${p}/root/$MOUNT_POINT/hello | MATCH "Hello World!"; then
```

Comment:
You can do this instead:
MATCH "Hello World!" < /proc/${p}/root/$MOUNT_POINT/hello